Ginger/GingerCoreNET/Drivers/CoreDrivers/Mobile/Appium/GenericAppiumDriver.cs

```diff
@@ -187,7 +188,8 @@ public bool ConnectToAppium()
                         }
                         break;
                 }
-                mSeleniumDriver = new SeleniumDriver(Driver); //used for running regular Selenium actions               
+                mSeleniumDriver = new SeleniumDriver(Driver); //used for running regular Selenium actions                                             
+                mSeleniumDriver.StartDriver();
```

Comment:
why to StartDriver? we only using it for it methodes reuse, we don't want to start another driver/browser

Reply:
It was mistakenly checked in, I was trying something as element detection was failing for AndroidBrowser